packages/test/test-utils/api-report/test-utils.api.md

```diff
@@ -138,6 +138,16 @@ export type fluidEntryPoint = SupportedExportInterfaces | IFluidModule;
 // @public (undocumented)
 export function getUnexpectedLogErrorException(logger: EventAndErrorTrackingLogger | undefined, prefix?: string): Error | undefined;
 
+// @public (undocumented)
```

Comment:
I would strongly recommend adding documentation to any of the new / modified items here that are annotated as `undocumented`.

Reply:
Looks like a lot of it is new implementations of existing interfaces, in which case we can leverage `{@inheritDoc Foo.bar}` for much of it.

Reply:
Just pushed a commit to update all the docs for both `TestObjectProvider` and `TestObjectProviderWithVersionedLoad`!

Reply:
Since `IDocumentIdStrategy` is publicly exported in the index.ts, I would assume that would need some documentation too, right?